src/script/view_model/WarningsContainer/WarningsContainer.tsx

```diff
@@ -154,7 +75,7 @@ const WarningsContainer: React.FC = () => {
   return (
     <div className={cx('warning', {'warning-dimmed': warningDimmed})}>
       {visibleWarning === type.REQUEST_CAMERA && (
-        <div className="warning-bar warning-bar-feature">
+        <div data-uie-name="request-camera" className="warning-bar warning-bar-feature">
```

Comment:
how about using `visibleWarning` everywhere in the `data-uie-name` so that we don't have to come up with further names when new warnings appear?

Reply:
we could even have a generic `<div>` wrapper that would set the uieName and the class according to the type of warning:

```jsx
const classnames: Record<type, string> = {
 [type.REQUEST_CAMERA]: 'warning-bar warning-bar-feature',
 [type.NO_INTERNET]: '...'
 //...
}

<div data-uie-name={visibleWarning} className={classnames[visibleWarning]}>
  { visibleWarning === type.REQUEST_CAMERA }
  { visibleWarning === type.NO_INTERNET }
  ...
</div>
```

Reply:
And now that you are backed by tests, you will know that you are not breaking the current feature ;)

Reply:
Ah shit, you might also need a mapping from `warningType => uieName` since we already have some uie-names in the wild that we need to restore :/ 

Nevermind ignore my previous comment, let's keep it the way you made it 👍